project4part2/presentation/overview.tex

```diff
@@ -9,11 +9,11 @@ \section{Implementation Overview}
 \begin{itemize}
 \item Our project consists of \code{C/C++} files as well
   as a \code{makefile}.
-\item As previous presented, it also includes several
-  libraries to help with cryptosystem functionality, the
-  main one being the GMP library.
+\item As previously mentioned, it also includes several
+  libraries (what are they?) to help with cryptosystem
+  functionality, the main one being the GMP library.
```

Comment:
We're using `GMP` as our only *uncommon* library (beside *common* libraries like `iostream` and `vector`). `Msieve` will unlikely be necessary.